### ISSUE TITLE
Conda/Pip install from github not working

### ISSUE THREAD

**Issue by rluedde** [cenpy-devs/cenpy#117]
[This](https://stackoverflow.com/questions/19042389/conda-installing-upgrading-directly-from-github) says you can install packages directly from Github into a conda environment.

`pip install git+https://github.com/cenpy-devs/cenpy@master` throws a 
```
Collecting git+https://github.com/cenpy-devs/cenpy@master
  Cloning https://github.com/cenpy-devs/cenpy (to revision master) to /tmp/pip-req-build-uy0cqot7
  Running command git clone -q https://github.com/cenpy-devs/cenpy /tmp/pip-req-build-uy0cqot7
Requirement already satisfied: pandas in ./anaconda3/envs/cenpy_dev/lib/python3.7/site-packages (from cenpy===-1.0.0post3-) (1.0.5)
Collecting geopandas
  Using cached geopandas-0.8.0-py2.py3-none-any.whl (962 kB)
Collecting requests
  Using cached requests-2.24.0-py2.py3-none-any.whl (61 kB)
Collecting rtree
  Using cached Rtree-0.9.4.tar.gz (62 kB)
    ERROR: Command errored out with exit status 1:
     command: /home/reilley/anaconda3/envs/cenpy_dev/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-in_jjwzd/rtree/setup.py'"'"'; __file__='"'"'/tmp/pip-install-in_jjwzd/rtree/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-wh5e2wf4
         cwd: /tmp/pip-install-in_jjwzd/rtree/
    Complete output (15 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-in_jjwzd/rtree/setup.py", line 3, in <module>
        import rtree
      File "/tmp/pip-install-in_jjwzd/rtree/rtree/__init__.py", line 1, in <module>
        from .index import Rtree
      File "/tmp/pip-install-in_jjwzd/rtree/rtree/index.py", line 6, in <module>
        from . import core
      File "/tmp/pip-install-in_jjwzd/rtree/rtree/core.py", line 143, in <module>
        rt.Error_GetLastErrorNum.restype = ctypes.c_int
      File "/home/reilley/anaconda3/envs/cenpy_dev/lib/python3.7/ctypes/__init__.py", line 377, in __getattr__
        func = self.__getitem__(name)
      File "/home/reilley/anaconda3/envs/cenpy_dev/lib/python3.7/ctypes/__init__.py", line 382, in __getitem__
        func = self._FuncPtr((name_or_ordinal, self))
    AttributeError: /home/reilley/anaconda3/envs/cenpy_dev/bin/python: undefined symbol: Error_GetLastErrorNum
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

I'd like to be able to have the `master` branch importable (instead of the latest release) when I'm working on making changes to cenpy or if I want to have some new feature. What's a better way to do this?

**Comment by knaaptime** [comment]
if you install geopandas and deps (rtree) using anaconda, you should be able to

**Comment by rluedde** [comment]
`python setup.py develop` throws a `ValueError: bad marshal data (unknown type code)`

However, I found that `pip install -e cenpy/` installs cenpy from the `develop` channel into the conda env. For anyone in the future, I ran this command from the the directory level above the one containing the `setup.py` file (after a `git clone`). I had also installed `rtree` and `geopandas` like @knaaptime recommended (thank you by the way).
